component/admin/jevents.php

```diff
@@ -30,17 +30,18 @@
 	return;
 }
 
-if (isset($_SERVER['HTTP_USER_AGENT']) && (strpos($_SERVER['HTTP_USER_AGENT'], "MSIE") !== false || strpos($_SERVER['HTTP_USER_AGENT'], "Internet Explorer") !== false))
+if (!class_exists('GSLMSIE10') && isset($_SERVER['HTTP_USER_AGENT']) && (strpos($_SERVER['HTTP_USER_AGENT'], "MSIE") !== false || strpos($_SERVER['HTTP_USER_AGENT'], "Internet Explorer") !== false))
```

Comment:
GSLMSIE10 is a constant not a class - should be 

`if (!defined('GSLMSIE10')`

But I don't see how it could be defined already - this is the entry point into the backend of JEvents

Reply:
Opps! I would assume a plugin maybe or the router is being called twice.